src/main/java/org/codeforamerica/messaging/models/Message.java

```diff
@@ -23,6 +23,8 @@ public class Message {
     @Id
     @GeneratedValue(strategy= GenerationType.IDENTITY)
     private Long id;
+    @ManyToOne
+    TemplateVariant templateVariant;
```

Comment:
@vrajmohan After spending some time exploring a refactor, I ended up keeping the `body` and `subject` at the `Messsage` level in addition to adding the `templateVariant`. I dislike the repetition of having these values duplicated in the `SmsMessage` and `EmailMessage`, but those object are currently only being created once they have been sent to a provider. I discovered the refactor was more than I was comfortable doing as part of this story, so I am leaving it as is. 

I also noticed that due to your message scheduling change the response no longer has the message objects attached. The refactor I'm picturing would allow for those to be returned with a "scheduled" status if there is interest in providing that.

Reply:
BTW, the response does not explicitly exclude the message objects. It is just that the POST returns without waiting for the send to happen. The GET endpoint will indeed show the message objects.